fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -360,8 +351,11 @@ public void sendVote(List<ElectionVote> votes) {
                   throwable -> Log.d(TAG, "timed out waiting for result on cast_vote", throwable));
 
       disposables.add(disposable);
-    } catch (GeneralSecurityException | IOException e) {
-      Log.d(TAG, PK_FAILURE_MESSAGE, e);
+    } else {
+      Log.i(
+          TAG,
+          "User tried to send a message that expected proof-of-personhood but it could not be achieved.");
+      Toast.makeText(getApplication(), "your PoPToken is invalid", Toast.LENGTH_LONG).show();
```

Comment:
is it invalid, or unavailable ? 🤔

Reply:
It depends.

I will rethink the interface because it might be good to differentiate between an uninitialized Wallet, an error in the generation and a token that is not present as attendees.